### PR TITLE
Wolfram command

### PR DESCRIPTION
**Pull request by Gilbert142**
**Please describe the changes this PR makes and why it should be merged:**
Port 4.0.2 wolfram command to V5

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [ ] This PR changes command functions
  - [ ] This PR changes metadata for the command, updated in `commands.js`
  - [x] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/wolfram.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+				color: Colors.INVALID,
+				description: Text.INVALID_USAGE(commandData, msg.guild.commandPrefix),
+			},
+		});
+	}
+
+	const response = await msg.send(Text.THIRD_PARTY_FETCH("Fetching data from Wolfram|Alpha"));
+
+	const query = encodeURIComponent(msg.suffix);
+	const body = (await fetch.get(APIs.WOLFRAM(wolframAppID, query)).set("User-Agent", UserAgent).onlyBody()).queryresult;
```
Comment (vladfrangu) [inline Commands/Public/wolfram.js:17]: This can throw if api errors...

File: Commands/Public/wolfram.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	if (body.success && body.pods.length) {
+		const fields = body.pods.map(pod => ({
+			name: pod.title,
+			value: pod.subpods[0].plaintext && pod.subpods[0].plaintext !== "" ? pod.subpods[0].plaintext : `[Image](${pod.subpods[0].img.src})`,
```
Comment (vladfrangu) [inline Commands/Public/wolfram.js:21]: Should use the image prop

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: Commands/Public/wolfram.js (hunk 1 above)
Comment (Gilbert142) [reply Commands/Public/wolfram.js:17]: It should. It’ll get handled by the generic command handler error catcher. The message seemed fit for this type of error; unexpected and unknown.

File: Commands/Public/wolfram.js (hunk 2 above)
Comment (Gilbert142) [reply Commands/Public/wolfram.js:21]: One wolfram query could return multiple images

Comment (vladfrangu) [reply Commands/Public/wolfram.js:21]: You're only using the first image anyways, so what gives?

Comment (Gilbert142) [reply Commands/Public/wolfram.js:21]: We're using the image of the first subpod of every pod. Every pod still gets their own field, and each pod can have an image.

**Pull request merged**